client/client.go

```diff
@@ -814,7 +814,8 @@ func (ec *SDKClient) GetGasPrice(
 ) (*big.Int, error) {
 	var gasPrice *big.Int
 	var err error
-	if input.GasPrice == nil {
+	if input.GasPrice == nil || input.GasPrice.Uint64() == 0 {
+		log.Println("Fetching gas price")
```

Comment:
these logs needed or they are just for debugging?

Reply:
not just for debugging, added more logs for clarifying purposes